test/microbenchmarks-compare.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Please add the copyright header, and also

```suggestion
#!/usr/bin/env bash
```